### PR TITLE
Updated workshop schedule with 5 papers presenting

### PR DESCRIPTION
**Pull request by psteinb**
@oliverguhr @HeidiSeibold please check. We only have 5 papers that will present. So the timing for our workshop shifts considerably.

### CODE REVIEW

**Comment by psteinb** [comment]
more over, in order to comply with the conference schedule, we have to start at 9am CEST.

**Comment by HeidiSeibold** [comment]
Looks good to me. A bit more time will be good for the authors and if they are too short, it isn't a problem. Maybe we write 5-7 Minutes for those who expected to have 5 and planned for it already.

 Are we "allowed" to run the workshop until 1.45?

File: index.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ The central activity of the workshop will be twofold:
 
 1. a call-for-papers whereby teaching professionals or beginners are asked to describe their method of choice when teaching a given ML topic. We like to attract at maximum 4-page long mini-articles (excluding references and acknowledgements) that present or discuss a teaching activity related to machine learning. For more details, see below.
 
-2. **presentations of 5 minute lightning talks of all accepted papers** during the workshop at ECML PKDD 2020
+2. **presentations of 7 minute lightning talks of all accepted papers** during the workshop at ECML PKDD 2020
```
Comment (HeidiSeibold) [inline index.md:122]:
```suggestion
2. **presentations of 5-7 minute lightning talks of all accepted papers** during the workshop at ECML PKDD 2020
```

**Review by HeidiSeibold — CHANGES_REQUESTED** [review]

**Comment by psteinb** [comment]
About the end of the workshop, I don't know. As it is virtual and we'll write a paper potentially and have an etherpad of sorts, I hope that it doesn't matter.

**Comment by psteinb** [comment]
Ok, I'll merge now.

**Pull request merged**